app/controllers/users_controller.rb

```diff
@@ -20,8 +17,7 @@ def show
     @item = Item.new
   end
 
-  def index
```

Comment:
@fuzzyalej Yeah, I mistakenly committed to a different branch, but here's the right commit https://github.com/ivorpad/blocitoff/commit/182f48740ad19df86be5118bf70d5e171809b7cb